go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/jimschubert/docked
 
-go 1.17
+go 1.22
```

Comment:
This should not change. It defines the minimum version required for the module, which should not be changed as it impacts all projects which impact this.

While pkg.go.dev may not show imports, I do use this as work and can't upgrade projects to 1.22 yet.

Reply:
reverted